README.md

```diff
@@ -1,39 +1,62 @@
-<p align="center">
-  <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png" />
-</p>
```

Comment:
@lydiasama @chornerman I am not sure about removing the header logo and steps to set up a new project from the template 🤔

Reply:
@hoangnguyen92dn I Bring back all the parts from the Android template and stuck my part under it. Please let me know if you have more suggestions.

Reply:
@chornerman 🤔  This repo is for Github Action runner so I think you could put it on top of the step-up steps. Both are good to me 😄